code/game/area/Space Station 13 areas.dm

```diff
@@ -435,7 +435,8 @@ proc/process_adminbus_teleport_locs()
 /area/shuttle/vox/station
 	name = "\improper Vox Skipjack"
 	icon_state = "yellow"
-	requires_power = 0
```

Comment:
wtf i removed this? oops

Reply:
actually. it looks like /area/shuttle has requires_power=0 on it anyways so i don't think this would've actually done anything. BUT JUST TO BE SAFE.